### DISCUSSION TITLE
Wireguard только для списка zapret

### DISCUSSION THREAD

**Discussion by bazaev** [bol-van/zapret#878] [Q&A]
Как настроить Wireguard таким образом, что бы клиентский трафик через него проходил только до заблокированных адресов? Нашёл только то что в клиентской конфиге можно задавать IP sets. Но что делать если их много и не все известны? Судя по `/opt/zapret/ipset/create_ipset.sh`, zapret генерирует адреса с которыми работает: можно ли это как либо использовать для текущей задачи? 

Сейчас wireguard настроен так, что весь клиентский трафик проходит через него. А в конфиге zapret'а включен `--desync-any-protocols`

**Comment by bol-van** [comment]
2 варианта. Или руками отдельно от скриптов запуска, или custom script.

**Comment by bazaev** [comment]
> custom script

то есть при каждом старте `zapret` генерировать правила для `Wireguard`?

**Comment by bol-van** [comment]
Почитайте в доке что такое custom script.
